ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -1104,23 +1165,34 @@ const DuccCard = (props: {
   activeModule: AccessModule;
   clickableModules: AccessModule[];
   spinnerProps: WithSpinnerOverlayProps;
+  pageMode: DARPageMode;
   stepNumber: number;
 }) => {
-  const { profile, activeModule, clickableModules, spinnerProps, stepNumber } =
-    props;
+  const {
+    profile,
+    activeModule,
+    clickableModules,
+    spinnerProps,
+    pageMode,
+    stepNumber,
+  } = props;
   return (
-    <FlexRow style={{ ...styles.card, height: '125px' }}>
+    <FlexRow style={styles.card}>
```

Comment:
Does the height not to be set anymore?

Reply:
Right.  The components now take up whatever height is necessary.  For the old version that's a minor change (but still an improvement I think) but for this new version it's a necessity because the cards take up much more space.